hostscripts/clouddata/syncSLErepos

```diff
@@ -32,6 +33,7 @@ elif [ $SITE = "prv" ]; then
     # some stuff not available in the provo ibs mirror and it would
     # then disappear
     rsync_delete=""
+    rsync_excludes="--exclude=*src.rpm --exclude=*-untested.qcow2 --exclude=-debugsource*.rpm --exclude=-debuginfo*.rpm"
```

Comment:
debugsource and debuginfo globs need a leading `*` - and you could also keep the 2nd minus
--exclude=*-debuginfo-*.rpm

Reply:
maybe move the src.rpm into a 'common exclude' place to avoid duplication - or just keep it in the old place